app/serializers/api/v2/measures/measure_type_serializer.rb

```diff
@@ -8,10 +8,18 @@ class MeasureTypeSerializer
 
         set_id :measure_type_id
 
-        attributes :description, :national, :measure_type_series_id
+        attributes :description,
+                   :measure_type_series_id,
+                   :measure_component_applicable_code,
+                   :order_number_capture_code,
+                   :trade_movement_code,
+                   :validity_end_date,
+                   :validity_start_date
 
-        attribute :id do |measure_type|
-          measure_type.measure_type_id
+        attribute :id, &:measure_type_id
+
+        attribute :measure_type_series_description do |measure_type|
```

Comment:
The pattern we use elsewhere is to have this description delegator on the model.